[mypy]

[mypy-pymonzo]
ignore_errors = True

[mypy-pymonzo.monzo_api]
ignore_errors = True

# TODO hmm. fix this?
# pymonzo/api_objects.py:17: error: Need type annotation for '_required_keys' (hint: "_required_keys: List[<type>] = ..."
#
[mypy-pymonzo.api_objects]
ignore_errors = True
